src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -271,6 +271,38 @@ public void onInstallShiny(InstallShinyEvent event)
                 new Command() { public void execute() {}});
    }
    
+   public void withDataImport(String userAction, final Command command)
```

Comment:
Good point, I'm changing it not to be CSV specific. I will add additional dependencies as needed.